iap/README.md

```diff
@@ -0,0 +1,39 @@
+# Cloud Identity-Aware Proxy Java Samples
+Cloud Identity-Aware Proxy (Cloud IAP) lets you manage access to applications running in Compute Engine, App Engine standard environment, and Container Engine. Cloud IAP establishes a central authorization layer for applications accessed by HTTPS, enabling you to adopt an application-level access control model instead of relying on network-level firewalls. When you enable Cloud IAP, you must also use signed headers or the App Engine standard environment Users API to secure your app.
+
+## Setup
+- A Google Cloud project with billing enabled
+- [Create an App engine service account](https://cloud.google.com/docs/authentication#getting_credentials_for_server-centric_flow) and download the credentials file as JSON.
+- Install the [Google Cloud SDK](https://cloud.google.com/sdk/) and run:
+```
+   gcloud init
+   gcloud app create
```

Comment:
`gcloud app create` required if its the first time you're creating an App Engine application.
Added comment stating that.
Authentication not required here, since it cannot be run locally.